### ISSUE TITLE
Re: A GUI interface

### ISSUE THREAD

**Issue by tripleo1** [40uf411/SemaWal#4]
https://github.com/Wanderson-Magalhaes/Modern_GUI_PyDracula_PySide6_or_PyQt6

I don't know QT as of this writing, but this sure looks nice...

**Comment by 40uf411** [comment]
Indeed, I am working on a new version of the library. Perhaps it will come with GUI.
Thanks for the recommendation.
